### PR TITLE
Translate Contributing in README and add contributing guide

### LINKED ISSUE #28: Traduzir a seção Contributing do README.md para PT-BR
No `README.md`, a seção **Contributing** está em Inglês, sendo que este é um repositório com a tradução da documentação de um repo para PT-BR, ou seja, a galera que vai contribuir com traduções devem falar português, portanto, faz mais sentido que esta seção esteja em PT-BR também.

### PR DESCRIPTION
**Pull request by pedrohyvo**
Translate Contributing in README and add contributing guide.
Closes #28

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Para os novos contribuidores
+
+1. Faça um fork do projeto.
+2. Crie uma nova branch com as suas alterações: 
+3. `git checkout -b nome-pagina-nome-secao` (altere nome-pagina-nome-secao de acordo com o 
+nome da página ou seção que você está traduzindo).
+4. Salve as alterações e crie uma mensagem de commit contando o que você fez:
+`git commit -m "doc: Add Example page"`.
+5. Envie as suas alterações: `git push origin nome-pagina-nome-secao`.
+6. Verifique se a sua branch está atualizada com a branch `master` e submeta seu pull request!
```
Comment (wlsf82) [inline CONTRIBUTING.md:12]:
```suggestion
6. Verifique se a sua branch está atualizada com base no repositório upastream e submeta seu pull request!
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+**Pré-requisito: ter o node instalado na sua máquina.**
+
+Após fazer o fork do projeto no seu repositório local você deve rodar:
+
+```bash
+npm install
+node_modules/.bin/markdownlint ./ --ignore node_modules
+```
+
+obs: para os usuários do VSCode, existe uma extensão [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint).
```
Comment (wlsf82) [inline CONTRIBUTING.md:31]:
```suggestion
Obs: para os usuários do VSCode, existe uma extensão [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint).
```

**Review by wlsf82 — CHANGES_REQUESTED** [review]
Hi, @pedrohyvo,

I've left a few suggestions for you to apply.

After that, I'm ok with having this PR merged.

File: CONTRIBUTING.md (hunk 1 above)
Comment (lucastfa-toptal) [reply CONTRIBUTING.md:12]: typo: upstream

Comment (lucastfa-toptal) [inline CONTRIBUTING.md:12]: I suggest to add command examples for step 6 as you did for 3, 4 and 5.

**Review by lucastfa-toptal — COMMENTED** [review]
I left just a small suggestion.

File: CONTRIBUTING.md (hunk 1 above)
Comment (pedrohyvo) [reply CONTRIBUTING.md:12]: @lucastfa-toptal added now

**Review by lucastfa-toptal — APPROVED** [review]

**Pull request merged**